tests/conversion

```diff
@@ -255,7 +258,7 @@ IMAGE_PATH="${tmpdir}/backup/virtual-machine.img"
 # Test VM migration using conversion mode. If server does not support
 # conversion API extension, lxd-migrate must fallback to migration
 # mode and successfully transfer the VM disk.
-for driver in btrfs ceph dir lvm zfs; do
```

Comment:
Why is `ceph` removed?

Reply:
Cause I didn't bother setting up microceph for testing locally and accidentally scooped that edit in... thanks for catching!

Reply:
Oh incidentally, me accidentally dropping `ceph` from the tested storage driver probably explains why the tests were faster ... silly me!